pkg/descheduler/descheduler.go

```diff
@@ -398,7 +401,7 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 			// handle or function which the Evictor has access to. For migration/in-progress framework
 			// work, we are currently passing this via context. To be removed
 			// (See discussion thread https://github.com/kubernetes-sigs/descheduler/pull/885#discussion_r919962292)
-			childCtx := context.WithValue(ctx, "strategyName", pg.Name())
+			childCtx := context.WithValue(parentCtx, "strategyName", pg.Name())
```

Comment:
same question with these two, rather than creating a new `childCtx` this could just reassign to `ctx`. Though I also don't think you need to manually use `context.WithValue` here, but something like span attributes would be better